drivers/wifi/nrf700x/osal/fw_if/umac_if/src/cmd.c

```diff
@@ -169,6 +169,7 @@ enum wifi_nrf_status umac_cmd_init(struct wifi_nrf_fmac_dev_ctx *fmac_dev_ctx,
 #endif /* !CONFIG_NRF700X_RADIO_TEST */
 
 	umac_cmd_data->op_band = op_band;
+	umac_cmd_data->maxidle_insec = CONFIG_NRF700X_BSS_MAX_IDLE_TIME;
```

Comment:
Do we need to check the range here

Reply:
Putting a range restriction in Kconfig should be enough. That blocks the build if an out-of-range default value is configured in Kconfig.